src/ReflectionMethod.php

```diff
@@ -80,7 +80,8 @@ public function ___debugInfo()
      */
     public function __toString()
     {
-        $hasReturnType    = $this->hasReturnType();
+        $returnType       = $this->getReturnType();
+        $hasReturnType    = ($returnType !== null); // Internally same as: $this->hasReturnType();
```

Comment:
Adding the parentheses was inadvertent and I'll remove them, but I **do** think the comment is helpful. It's an explanation why we aren't calling `hasReturnType()`. @lisachenko seems to be deferring to you on this PR, so it's your call, but I think the comment has value.

if you disagree, I'll remove it; or if you prefer, I can put the comment on it's own line.